### PR TITLE
Fallback with connectionStatus when listDatabases is not authorized

### PR DESCRIPTION
**Pull request by okhowang**
## Description

Just like `show dbs` in mongo-cli.

### Checklist
- [x] New tests and/or benchmarks are included
- [ ] Documentation is changed or added

## Motivation and Context
when user has no privilege to execute listDatabases, it also can get dbs in UI

- [ ] Bugfix
- [x] New feature
- [ ] Dependency update
- [ ] Misc

## Types of changes

- [ ] *Backport Needed*
- [ ] Patch (non-breaking change which fixes an issue)
- [x] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by alenakhineika** [comment]
@okhowang we will be glad to merge it since it brings value to your working process. Before we can do it, please sing our contribution agreement:
https://github.com/mongodb-js/compass/blob/master/CONTRIBUTING.md

**Comment by okhowang** [comment]
done

**Comment by okhowang** [comment]
done

File: lib/instance-detail-helper.js [hunk 1]
```diff
@@ -264,9 +264,33 @@ function listDatabases(results, done) {
...
-        done(null, []);
+        // eslint-disable-next-line no-shadow
+        adminDb.command({connectionStatus: 1, showPrivileges: 1}, {}, function(err, res) {
+          if (err) {
+            done(err);
+            return;
+          }
+          const privileges = (res.authInfo || {}).authenticatedUserPrivileges;
+          if (privileges === undefined) {
+            done(null, []);
```
Comment (alenakhineika) [inline lib/instance-detail-helper.js:275]: Just noticed, add return here to make sure that the callback after if statement won't be called.

Comment (okhowang) [reply lib/instance-detail-helper.js:275]: Fixed

**Comment by alenakhineika** [comment]
`mongodb-data-service@16.8.0` was published 📦

@okhowang thank you for contributing!

**Pull request merged**